docker-compose.yml

```diff
@@ -1,5 +1,49 @@
 version: "3"
 services:
+  celestia-devnet:
+    container_name: celestia_devnet
+    image: ghcr.io/rollkit/local-celestia-devnet:v0.11.0-rc12
+    restart: on-failure
+    ports:
+      - "26657"
+      - "26658"
+      - "26659"
+      - "9090"
+
+  evm:
+    container_name: evm_base_shard
+    environment:
+      - DA_AUTH_TOKEN=${DA_AUTH_TOKEN:-}
+      - VALIDATOR_NAME=${VALIDATOR_NAME:-}
+      - CHAIN_ID=${CHAIN_ID-}
+      - KEY_NAME=${KEY_NAME:-}
+      - KEY_MNEMONIC=${KEY_MNEMONIC:-}
+      - KEY_BACKEND=${KEY_BACKEND:-}
+      - TOKEN_AMOUNT=${TOKEN_AMOUNT:-}
+      - STAKING_AMOUNT=${STAKING_AMOUNT:-}
```

Comment:
Why do we have staking

Reply:
ask rollkit 🤷🏼‍♂️

Reply:
im not sure what it does. can't we just set it to 0? idt we even need to surface this in the config, it will just confuse people

Reply:
it has to be >0. otherwise chain will not run. 
see panic message:

```
failed to execute message; message index: 0: invalid delegation amount: invalid request
```

Reply:
its rather unfortunate that rollkit is a hodgepodge of sort-of-PoS-chain, and sort-of-rollup..

Reply:
let's just it to 1 then lol